tests/test_hodgkin.py

```diff
@@ -240,3 +242,21 @@ def test_evaluation(self, graph, model):
             expected = evaluated_derivatives[state_symbol.name]
             actual = evaluated_deriv
             assert float(actual) == pytest.approx(expected)
+
+    def test_get_symbol_by_ontology_term(self, graph, model):
+        membrane_voltage_var = model.get_symbol_by_ontology_term(OXMETA, "membrane_voltage")
+        assert (isinstance(membrane_voltage_var, sympy.symbol.Dummy))
```

Comment:
You don't need () around assertions; similarly below.

Reply:
ok, fixed